CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-Welcome! Thanks for your interest in contributing to Globalize. You're **almost** in the right place. More information on how to contribute to this and all other jQuery Foundation projects is over at [contribute.jquery.org](http://contribute.jquery.org). You'll definitely want to take a look at the articles on contributing [code](http://contribute.jquery.org/code).
+Welcome! Thanks for your interest in contributing to Globalize. You're **almost** in the right place. More information on how to contribute to this and all other jQuery Foundation projects is over at [contribute.jquery.org](http://contribute.jquery.org). To help you get started and before you jump into writing code, be sure to read [Writing Code for jQuery Foundation Projects](http://contribute.jquery.org/code/).
```

Comment:
With this change, I wanted to emphasize one should get started with "Writing Code for jQuery Foundation Projects".